scripts/waypoints_test.py

```diff
@@ -49,6 +54,32 @@ def constrain(x, l, h):
             velocity.twist.linear.y = constrain((target[1] - trans[1]) * kP, -max_vel, max_vel)
         if abs(target[2] - trans[2]) >= 0.02:
             velocity.twist.linear.z = target[2] - trans[2]
+        
+        # Get the yaw (z axis) rotation from the quanternion
+        ysqr = rot[1] * rot[1];
```

Comment:
However I will change it here, as it will be more readable and there are no speed requirements.